src/pages/plugins/concepts/entry-points/index.md

```diff
@@ -0,0 +1,15 @@
+# UXP Entry Points
+
+Plugins can have multiple entry points. An entrypoint is a way for a plugin to be invoked.
+
+There are two types of entry points: `commands` and `panels`. 
+
+- Commands are _"run and forget"_ entry points that are invoked by the user, such as a menu item or a keyboard shortcut. 
```

Comment:
@anoopvalomkot Will we have support for keyboard shortcuts?

Reply:
I believe this needs development on InDesign side. If yes, it isn't supported.